shared/build.gradle.kts

```
@@ -1,10 +1,14 @@
 import org.jetbrains.kotlin.gradle.plugin.mpp.NativeBuildType
+import org.jetbrains.kotlin.konan.properties.loadProperties
+import com.codingfeline.buildkonfig.compiler.FieldSpec.Type.STRING
 
 plugins {
     kotlin(Plugins.MULTIPLATFORM)
     kotlin(Plugins.IOS_COCOAPODS)
     id(Plugins.ANDROID_LIBRARY)
     id(Plugins.KOVER)
+    id(Plugins.KSP) version (Versions.KSP)
```

please set global version for KSP in the root gradle > plugins block

```suggestion
    id(Plugins.KSP)
```